posts/2020/10/manylinux1-manylinux2010.md

```diff
@@ -0,0 +1,83 @@
+<!--
+.. title: Manylinux1 is obsolete, manylinux2010 is almost EOL, what is next?
+.. slug: manylinux1-is-obsolete-manylinux2010-is-almost-eol-what-is-next
+.. date: 2020-10-08 14:00:06 UTC-05:00
+.. tags: 
+.. category: 
+.. link: 
+.. description: 
+.. type: text
+-->
+
```

Comment:
It would be good to have another paragraph with a more gentle introduction here. Many readers won't know what manylinux and glibc really are. Something like:
- Wheels are binary distributions of Python packages, that you normally install with Pip from PyPI.
- For quite a while we had Windows and macOS wheels only.
- People wanted Linux binaries too, but due to different build toolchains (intro glibc) this was harder.
- In 201x the manylinux standard was therefore introduced (later renamed to `manylinux1`).

Reply:
reworked the teaser, but now I think it is too long.